app/localprojectsmanager.cpp

```diff
@@ -28,6 +28,16 @@ static QString findQgisProjectFile( const QString &projectDir )
   {
     return foundProjectFiles.first();
   }
+  else if ( foundProjectFiles.count() > 1 )
+  {
+    // error: multiple project files found
+    err = QString( "ERR_MULTIPLE_PROJECTS" );
```

Comment:
just put the translated error message directly here?

Reply:
Sure, we can.

Reply:
maybe not